pywbem/_logging.py

```diff
@@ -32,157 +34,206 @@
-  methods that drive WBEM operations (see :ref:`WBEM operations`) and their
-  responses before they are passed back to the user. This logs the parameters
-  of each request and the results including CIM objects/exceptions in each
-  method response. It also logs the creation of the
-  :class:`~pywbem.WBEMConnection` object to capture connection information.
-
-* ``pywbem.http`` - Logs HTTP requests and responses between the pywbem client
-  and WBEM server. This logs the http request data and response data including
-  http headers and XML payload.
-
-The above loggers output log records when they are configured and
-activated. The pywbem loggers log at the DEBUG log level.
-
-There are three components to configuration of the pywbem loggers:
-
-* **Configure Python logger names** - Sets the Python logger parameters
-  (logging handlers, message formats, etc.). Either Python logging
-  methods or methods defined in :ref:`Logging configuration methods` may be
-  used to configure the loggers. By default, the two pywbem loggers have null
-  handlers (see logging.NullHandler) configured so that logging is not
-  propagated up the Python logger hierarchy
-
-* **Set the log record detail level for pywbem loggers(Optional)** - Because
-  pywbem loggerss can generate large quantities of information, they allow
-  the user to control the quantity of information in each log record (i.e. all
-  data, summary information, or data to a maximum log length).  This is a pywbem
-  feature so it can only be configured using the pywbem methods defined in
-  :ref:`Logging configuration methods`.
-
-* **Activate the pywbem loggers** - A pywbem logger is activated to
-  actually output log records when any of the WBEMConnection methods that
-  communicate with a WBEM server are executed for that WBEMConnection object.
-  Activation is controlled by a parameter of the methods defined in
-  :ref:`Logging configuration methods`.
-
-.. _`Logging configuration methods`:
-
-Logging configuration methods
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-The pywbem loggers may be configured through the following pywbem methods. These
-methods must be used to set the log detail_level and activate the loggers.
-
-  * :meth:`~pywbem.WBEMConnection.configure_logger` - Configures and
-    optionally activates one or both loggers. If the logger name is 'all'
-    it acts on both loggers.
-
-  * :func:`configure_loggers_from_string` - Allows configuring and
-    activating the pywbem loggers from a single formatted string that defines
-    logger names, log destination, and detail level. This is most
-    useful in defining the loggers from a command line tool such as pywbemcli
-    so the complete logger configuration can be compressed into a single
-    command line string.
+Pywbem logging uses two Python :class:`~py:logging.Logger` objects which are
+termed `pywbem loggers`:
+
+* API logger (Python logger name: `'pywbem.api'`) - Logs API calls and returns,
```

Comment:
These names are only useful if the user uses the Python logger configure methods.

Reply:
I mentioned them exactly for that reason.